config.yml.default

```diff
@@ -58,81 +58,81 @@ engine:                      # Engine settings.
+fake_think_time: false             # Artificially slow down the bot to pretend like it's thinking.
+rate_limiting_delay: 0             # Time (in ms) to delay after sending a move to prevent "Too Many Requests" errors.
+move_overhead: 2000                # Increase if your bot flags games too often.
 
 correspondence:
-  move_time: 60            # Time in seconds to search in correspondence games.
-  checkin_period: 600      # How often to check for opponent moves in correspondence games after disconnecting.
-  disconnect_time: 300     # Time before disconnecting from a correspondence game.
-  ponder: false            # Ponder in correspondence games the bot is connected to.
-
-challenge:                   # Incoming challenges.
-  concurrency: 1             # Number of games to play simultaneously.
-  sort_by: "best"            # Possible values: "best" and "first".
-  accept_bot: false          # Accepts challenges coming from other bots.
-  only_bot: false            # Accept challenges by bots only.
-  max_increment: 180         # Maximum amount of increment to accept a challenge. The max is 180. Set to 0 for no increment.
-  min_increment: 0           # Minimum amount of increment to accept a challenge.
-  max_base: 315360000        # Maximum amount of base time to accept a challenge. The max is 315360000 (10 years).
-  min_base: 0                # Minimum amount of base time to accept a challenge.
-  max_days: 14               # Maximum number of days per move to accept a challenge for a correspondence game.
-                             # Unlimited games can be accepted by removing this field or specifying .inf
-  min_days: 1                # Minimum number of days per move to accept a challenge for a correspondence game.
-  variants:                  # Chess variants to accept (https://lichess.org/variant).
+  move_time: 60                    # Time in seconds to search in correspondence games.
+  checkin_period: 300              # How often to check for opponent moves in correspondence games after disconnecting.
+  disconnect_time: 150             # Time before disconnecting from a correspondence game.
+  ponder: false                    # Ponder in correspondence games the bot is connected to.
+
+challenge:                         # Incoming challenges.
+  concurrency: 1                   # Number of games to play simultaneously.
+  sort_by: "best"                  # Possible values: "best" and "first".
+  accept_bot: true                 # Accepts challenges coming from other bots.
+  only_bot: false                  # Accept challenges by bots only.
+  max_increment: 20                # Maximum amount of increment to accept a challenge in seconds. The max is 180. Set to 0 for no increment.
+  min_increment: 0                 # Minimum amount of increment to accept a challenge in seconds.
+  max_base: 1800                   # Maximum amount of base time to accept a challenge in seconds. The max is 315360000 (10 years).
```

Comment:
Change the comment `The max is 315360000 (10 years).` to `The max is 10800 (3 hours).`

Reply:
Done.